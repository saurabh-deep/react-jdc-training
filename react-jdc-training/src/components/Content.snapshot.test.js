import React from "react";
import renderer from 'react-test-renderer';
import Content from "./Content";

describe("Content", () => {
  const contentText = "Testing content component text";

  it("properly shows given text", () => {
    const rendered = renderer.create(
      <Content text={contentText} />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});