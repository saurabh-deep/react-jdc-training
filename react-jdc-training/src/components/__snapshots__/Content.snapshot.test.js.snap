// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content properly shows given text 1`] = `
<p
  className="App-intro"
>
  Testing content component text
</p>
`;
